coderunners/compilers.py

```diff
@@ -54,3 +56,26 @@ def compile(self, submission_path: Path):
         binary_path.unlink(missing_ok=True)
         executable_path = f'{self.language_standard} {submission_path}'
         return executable_path, compile_res
+
+@dataclass
+class CSharpCompiler(Compiler):
+    language_standard: str
+    supported_standards = {'c#'}
+
+    dotnet_path = Path('/root/.dotnet/dotnet')
+    project_file_path = Path('/tmp/program/program.csproj')
+    code_path = Path(project_file_path.parent, 'Program.cs')
+    dll_path = Path('/tmp/out/program.dll')
+
+    def compile(self, submission_path: Path):
+        print('Creating CSharp project at:', self.project_file_path)
+        Process(f'{self.dotnet_path} new console -o {self.project_file_path.parent}' + f'&& cat {submission_path} > {self.code_path}',
```

Comment:
Would probably be better to use Pathlib rename like `my_file.rename(to_file)` instead of `cat > `